app/_data/docs_nav_kgo_1.4.x.yml

```diff
@@ -103,6 +103,22 @@ items:
             url: /guides/upgrade/data-plane/rolling/
           - text: Blue / Green Deployment
             url: /guides/upgrade/data-plane/blue-green/
+      - text: Managing Konnect entities
+        items:
+          - text: Architecture overview
+            url: /guides/konnect-entities/architecture/
```

Comment:
For some reason this doesn't render at: https://deploy-preview-8084--kongdocs.netlify.app/gateway-operator/unreleased/guides/konnect-entities/architecture/

😢 

@kong-docs can you help? The build seems to be happy.

Reply:
I believe it's a matter of a base branch used in the PR. I temporarily switched it to `main`, forced pushed an amended commit and it's rebuilding now. Apparently, only PRs targeting `main` have the Netlify preview rendered.

Reply:
Oh, hidden secret ㊙️ Thanks for the hint!

Reply:
Solved by @czeslavo by retargeting the branch to `main` temporarily.